src/component/inspect-request/request-headers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const {object} = React.PropTypes;
+
+const RequestHeaders = (props) => {
```

Comment:
Is it really a `RequestHeader`? Though? What about when it's query parameters? or response headers?
I mean, it's better than `KeyValue`, which is so generic it hurts.
Just saying, if a better name comes to mind, let's do it